routes/transactionBreakdown.ts

```diff
@@ -0,0 +1,110 @@
+import { Request, Response } from 'express';
+import express from 'express';
+const router = express.Router();
+
+const createItemList = (items: Array<any>) => {
+  return items.map((individual: any, ind: number) => {
+    const indItem = {
```

Comment:
if we remove the brackets, we don't need to create a variable and return:
```
const createItemList = (items: Array<any>) => {
   return items.map((individual: any, ind: number) => {
      id: ind,
      name: individual.name,
      people: individual.people,
      cost: individual.cost,
   });
}
```

Reply:
Hm... maybe this doesn't work lol. Lemme see...

Reply:
Maybe define an items interface:
```
interface Item {
  id: Number;
  name: String;
  people: Array[String];
  cost: Number;
}

const createItemList = (items: Array<any>) => {
  return items.map((item: any, index: number) => {
    id: index,
    name: item.name,
    people: item.people,
    cost: item.cost
  } as Item);
}
```

Reply:
Also, wait, what's the point of this method again? Are we just recreating the item list and assigning an index?

Reply:
Yea actually that is basically what it's doing atm. I was thinking maybe to leave it for when we finally do save to DB but im not sure if when you save to db does a unique id get created already?

Reply:
yea the id is associated with the db entry so it should be auto populated.

Reply:
Okay then in that case, I'll just forgoe this function since id isn't even really used